app/matplotlib.ipynb

```diff
@@ -129,7 +140,13 @@
     "categories = [\"Category A\", \"Category B\", \"Category C\", \"Category D\"]\n",
     "sales = [350, 480, 210, 590]\n",
     "\n",
-    "# write your code here\n"
+    "plt.bar(categories, sales)\n",
+    "\n",
+    "plt.title(\"Sales by Category\")\n",
+    "plt.xlabel(\"Categories\")\n",
+    "plt.ylabel(\"Sales\")\n",
+    "\n",
+    "plt.xticks(rotation=45)\n"
```

Comment:
You should add `plt.show()` at the end of this cell to ensure the bar plot is displayed in the notebook.